src/Helper/Arr.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the PHP-CLI package.
+ *
+ * (c) Jitendra Adhikari <jiten.adhikary@gmail.com>
+ *     <https://github.com/adhocore>
+ *
+ * Licensed under MIT license.
+ */
+
+namespace Ahc\Cli\Helper;
+
+use function current;
+use function func_get_args;
+use function is_array;
+use function is_int;
+use function next;
+
+/**
+ * helper specializing in table manipulation
+ *
+ * @author  Dimitri Sitchet Tomkeu <devcode.dst@gmail.com>
+ * @license MIT
+ *
+ * @link    https://github.com/adhocore/cli
+ */
+class Arr
```

Comment:
can we remove this helper at all?

Reply:
It's true that we don't necessarily need him. the initial idea was that it could be used later when we had problems in this direction. but thinking about it, I think we can do without it